src/__tests__/fetchInvolvementAPI.test.js

```diff
@@ -0,0 +1,25 @@
+/**
+ *@jest-environment jsdom
+ */
+
+import * as Involve from '../modules/fetchInvolvementAPI.js';
+import { likeslist, mealList, commentList } from '../__mocks__/fetchApi.js';
+
+test('count likes for certain item', () => {
+  const likes = likeslist();
+  const itemId = '52874';
+  const count = Involve.countLike(likes, itemId);
+  expect(count).toBe(4);
+});
+
+test('count items', () => {
+  const meals = mealList();
+  const count = meals.length;
+  expect(count).toBe(4);
+});
+
+test('count comments', () => {
+  const comments = commentList();
+  const count = comments.length;
+  expect(count).toBe(3);
+});
```

Comment:
- [ ] you are expected to create a specific function to calculate counter numbers on each page, at this moment there are no separate functions for the meal list and the comment list. please include this.
- [ ] after creating the above-mentioned functions make sure they are properly tested as well.